pkg/kubelet/hyper/hyperclient.go

```diff
@@ -506,12 +338,15 @@ func (c *HyperClient) PullImage(image string, credential string) error {
 		}
 	}
 
+	ctx, cancel := getContextWithTimeout(hyperContextTimeout)
```

Comment:
We can not expect how long it will take to pull image, maybe do not use timeout ctx here?